File: 프로그래머스/2/42584. 주식가격/주식가격.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> prices) {
    vector<int> answer;
    
    for (int i = 0; i < prices.size(); i++) {
        int count = -1;
        for (int j = i; j < prices.size(); j++) {
            count++;
            if (prices[i] > prices[j])
                break;
        }
        answer.push_back(count);
    }
    return answer;
}

File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = []
    n = len(prices)
    for i in range (0, n-1):
        count = 0
        for j in range(i+1, n):
            count += 1
            if prices[i] > prices[j]:
                break
        answer.append(count)
    answer.append(0)
    return answer